agency_api/api.py

```diff
@@ -299,3 +299,97 @@ def retrieve(self, request, pk):
         serializer = self.serializer_class(service_request, many=False)
 
         return Response(serializer.data)
+
+class HPJobApplicationViewSet(viewsets.ModelViewSet):
+    #permission_classes = [CustomModelPermissions]
+    serializer_class = HPJobApplicationRetrieveSerializer
+    #http_method_names = ['get', 'post']
+
+    def get_queryset(self):
+        return HPJobApplication.objects.filter(is_pending=True)
+
+    # GET /api/caretaker_requests
```

Comment:
Can we update these comments to match what the actual end point is? I think this one is /api/job_advertisement_request